### PR TITLE
Updates for smoke tests & deploy.sh

### PR DESCRIPTION
**Pull request by Atomic182**
- Added persistent volume for reports & logs
- Added removal of old(7+ days) reports & logs
- Added IDM live check

### CODE REVIEW

File: bin/deploy.sh [hunk 1]
```diff
@@ -106,7 +106,8 @@ chk_config()
     fi
     echo -e "=>\tComponents: \"${COMPONENTS[*]}\""
 
-    AM_URL="${URL_PREFIX:-openam}.${NAMESPACE}.${DOMAIN}" 
+    AM_URL="${URL_PREFIX:-openam}.${NAMESPACE}.${DOMAIN}"
+    IDM_URL="${IDM_URL_PREFIX:-openidm}.${NAMESPACE}.${DOMAIN}"
```
Comment (wstrange) [inline bin/deploy.sh]: looks good. This might shortly change in the future to a new 'login' prefix - but good for now

File: bin/deploy.sh [hunk 2]
```diff
@@ -183,6 +184,29 @@ isalive_check()
     echo "=> AM is alive"
 }
 
+isalive_check_idm()
+{
+    PROTO="https"
+    if [[ -f ${CFGDIR}/openidm.yaml || -f ${CFGDIR}/common.yaml ]]; then
+        if $(grep -sq -e '^tlsStrategy:\s*\bhttp\b' ${CFGDIR}/common.yaml ${CFGDIR}/openidm.yaml); then
+            PROTO="http"
```
Comment (dgoldssfo) [inline bin/deploy.sh]: I was under the impression that you can no longer use tlsStrategy for http - or did we leave it in just for the smoke tests?

Currently it's an undocumented option.

Comment (Atomic182) [reply bin/deploy.sh]: Right! My bad, I've just copied whatever we had there before. I will remove this and we go with hardcoded https

Comment (Atomic182) [reply bin/deploy.sh]: Removed this for both am & idm

**Pull request merged**